.github/workflows/main.yml

```diff
@@ -28,6 +28,12 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - test_name: "lua5.4"
+            lua_version: "5.4"
+            lua_name: "lua5.4"
+            lua_include: "/usr/local/include"
```

Comment:
This is required because we manually build Lua 5.4.

"If" a package is available in the future (because Ubuntu add it to their repositories, or we can move the GitHub Actions to uses Ubuntu 21), this path will most probably required to be updated.

Reply:
Highly unlikely that we'll see Ubuntu 21 in GH Actions. They've only used LTS releases so far, and the next one will be Ubuntu 22.
And we won't see Lua 5.4 in Ubuntu 20 either, as releases don't get feature upgrades.